packages/patternfly-react/sass/patternfly-react/_compound-label.scss

```diff
@@ -25,7 +21,7 @@
   .list-inline {
     display: inline-block;
 
-    &>li {
+    & > li {
```

Comment:
this file shouldn't need to be included, rebasing should resolve the issues

Reply:
? I changed this file to get rid of eslint warnings during builds.

Reply:
it has been fixed already

Reply:
ok, rebased, it did go away -thanks